lib/rules/property-no-unknown/index.js

```diff
@@ -67,12 +70,16 @@ function rule(actual, options) {
 				return;
 			}
 
-			const { selector } = decl.parent;
+			const { selector, name } = decl.parent;
 
 			if (selector && optionsMatches(options, 'ignoreSelectors', selector)) {
 				return;
 			}
 
+			if (name && optionsMatches(options, 'ignoreAtRules', name)) {
```

Comment:
[suggestion] How about checking also `type`? Because:

- More clear of the code intention
- Safer even if other node types would support the `name` property.

```suggestion
			if (type === 'atrule' && optionsMatches(options, 'ignoreAtRules', name)) {
```

See also <https://postcss.org/api/#atrule-type>